Tests/KituraTests/TestRequests.swift

```diff
@@ -26,7 +26,8 @@ class TestRequests: XCTestCase {
         return [
                    ("testURLParameters", testURLParameters),
                    ("testCustomMiddlewareURLParameter", testCustomMiddlewareURLParameter),
-                   ("testCustomMiddlewareURLParameterWithQueryParam", testCustomMiddlewareURLParameterWithQueryParam)
+                   ("testCustomMiddlewareURLParameterWithQueryParam", testCustomMiddlewareURLParameterWithQueryParam),
+                   ("testParameters", testParameters)
```

Comment:
You can leave a trailing comma here so future tests can be added without modifying this line.